Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/Follow.cs

```diff
@@ -0,0 +1,573 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Solvers
+{
+    /// <summary>
+    /// Follow solver positions an element relative in front of the forward axis of the reference.
+    /// The element can be loosly constrained (a.k.a. tag-along) so that it doesn't follow until it is too far.
+    /// </summary>
+    public class Follow : Solver
+    {
+        [SerializeField]
+        [Tooltip("Position lerp multiplier")]
+        private float moveToDefaultDistanceLerpTime = 0.1f;
+
+        /// <summary>
```

Comment:
I've always found this comment / description unclear. Instead of describing what the variable is, it would be more helpful to explain how increasing / decreasing the value changes the follow behavior. Also, does the "time" here actually mean "seconds for object to lerp to the desired position"? I don't see how the math leads to that, I always wondered if "time" here was misleading...

Reply:
I really like the idea of tool tips explaining the mechanism of what happens. MRTK has a lot of tool tips that are just redundant of the parameter name instead of really illuminating and helpful to the developer